src/Templates/parts/table-content.twig

```diff
@@ -18,7 +18,7 @@
                 <a href="{{column.url}}" title="{{column.title}}"><span class="glyphicon glyphicon-{{column.value}}">&nbsp;</span></a>
         {% endif %}
       {% else %}
-                {{column | raw}}
+                {{column}}
```

Comment:
Yep, that would've been it, and that would be right. It comes back to the currently over-flexible TinyMCE